core/base/inc/ROOT/RConfig.hxx

```diff
@@ -571,6 +571,18 @@
 #endif
 #endif
 
+// See also https://nemequ.github.io/hedley/api-reference.html#HEDLEY_NEVER_INLINE
+// for other platforms.
+#ifdef R__HAS_ATTRIBUTE_NO_INLINE
+#define R__NEVER_INLINE inline __attribute__((noinline))
```

Comment:
`inline` should not be part of `R__NEVER_INLINE`

Reply:
it is part of always_inline ....

Reply:
Fair enough - I guess there's no use case for this without `inline` - or rather, any use case will happily accept having an `inline` keyword (i.e. even in-class method definitions)